libs/ng-mocks/src/index.ts

```diff
@@ -4,6 +4,7 @@ import './lib/common/ng-mocks-stack';
 import './lib/common/ng-mocks-global-overrides';
 
 export * from './lib/common/core.tokens';
+export * from './lib/common/func.is-standalone';
```

Comment:
It's an internal function and it shouldn't be a part of public interface. 

I think the test should be done in a different way. There are some examples in tests folder which verify different behaviors of different angular versions.

Reply:
@satanTime Thank you for your feedback. I've looked into the [tests](https://github.com/help-me-mom/ng-mocks/tree/master/tests) folder but couldn't find any tests that specifically cover the feature or logic related to Angular version checks. I only found some tests that skip execution based on the Angular version, like this:

```
if (Number.parseInt(VERSION.major, 10) < 14) {
  it('needs >=a14', () => {
    expect(true).toBeTruthy();
  });
  return;
}
```

However, I need to ensure code coverage for the Angular 19 version check within the `isStandalone` function.

I attempted to mock the `VERSION` object using `spyOnProperty`, but this wasn't possible as the `VERSION` object from `@angular/core` isn't configurable.

Given that `isStandalone` is an internal function, what do you think about moving the test file to the [common](https://github.com/help-me-mom/ng-mocks/tree/master/libs/ng-mocks/src/lib/common) folder? Please let me know if there are any other approaches I should consider.